client/starwhale/base/cloud.py

```diff
@@ -235,23 +235,31 @@ def _fetch_bundle_all_list(
         uri_typ: str,
         page: int = DEFAULT_PAGE_IDX,
         size: int = DEFAULT_PAGE_SIZE,
+        _filter: t.Dict[str, t.Any] = {},
```

Comment:
`_filter` -> `filters` ?

Reply:
In addition, the mutable default value as an argument is an anti-patten for python.

Reply:
done